patches/tModLoader/Terraria/ModLoader/IO/TileIO_Basic.cs

```diff
@@ -1,3 +1,4 @@
+using System;
```

Comment:
Figure out where call:
TileIO.LoadNewWorld

Reply:
Was unneeded. LoadEntries assigns CreateEntries off the get go, and unloadedEntryLookup is assigned to by builder.Build in LoadData before it is used.